src/components/playerstats/playerstats.js

```diff
@@ -210,6 +221,13 @@ define(['events', 'globalize', 'playbackManager', 'connectionManager', 'playMeth
             });
         }
 
+        if (mediaFileSize) {
+            sessionStats.push({
+                label: 'File size:',
```

Comment:
Can you also add this to the i18n files and call `globalize.translate()` here so it can be translated to other languages?